android-components/components/feature/prompts/src/main/res/values/strings.xml

```diff
@@ -112,29 +132,45 @@
+    <!-- Header for the select credit card prompt to allow users to fill a form with a saved credit card. -->
+    <string name="mozac_feature_prompts_select_credit_card_1">Use saved card</string>
+    <!-- Content description for expanding the select credit card options in the select credit card prompt. -->
+    <string name="mozac_feature_prompts_expand_credit_cards_content_description" moz:removedIn="123" tools:ignore="UnusedResources">Expand suggested credit cards</string>
     <!-- Content description for expanding the select credit card options in the select credit card prompt. -->
-    <string name="mozac_feature_prompts_expand_credit_cards_content_description">Expand suggested credit cards</string>
+    <string name="mozac_feature_prompts_expand_credit_cards_content_description_1">Expand saved cards</string>
     <!-- Content description for collapsing the select credit card options in the select credit prompt. -->
-    <string name="mozac_feature_prompts_collapse_credit_cards_content_description">Collapse suggested credit cards</string>
+    <string name="mozac_feature_prompts_collapse_credit_cards_content_description" moz:removedIn="123" tools:ignore="UnusedResources">Collapse suggested credit cards</string>
+    <!-- Content description for collapsing the select credit card options in the select credit prompt. -->
+    <string name="mozac_feature_prompts_collapse_credit_cards_content_description_1">Collapse saved cards</string>
+    <!-- Option in the expanded select credit card prompt that links to credit cards settings. -->
+    <string name="mozac_feature_prompts_manage_credit_cards" moz:removedIn="123" tools:ignore="UnusedResources">Manage credit cards</string>
     <!-- Option in the expanded select credit card prompt that links to credit cards settings. -->
-    <string name="mozac_feature_prompts_manage_credit_cards">Manage credit cards</string>
+    <string name="mozac_feature_prompts_manage_credit_cards_1">Manage cards</string>
     <!-- Text for the title of a save credit card dialog. -->
     <string name="mozac_feature_prompts_save_credit_card_prompt_title">Securely save this card?</string>
     <!-- Text for the title of an update credit card dialog. -->
     <string name="mozac_feature_prompts_update_credit_card_prompt_title">Update card expiration date?</string>
     <!-- Subtitle text displayed under the title of the save credit card dialog. -->
-    <string name="mozac_feature_prompts_save_credit_card_prompt_body">Card number will be encrypted. Security code won’t be saved.</string>
+    <string name="mozac_feature_prompts_save_credit_card_prompt_body" moz:removedIn="123" tools:ignore="UnusedResources">Card number will be encrypted. Security code won’t be saved.</string>
+    <!-- Subtitle text displayed under the title of the save credit card dialog. -->
+    <string name="mozac_feature_prompts_save_credit_card_prompt_body_1">Firefox encrypts your card number. Your security code won’t be saved.</string>
```

Comment:
I don't think we want to include "Firefox" here and do what we typically do with using a `%s` and filling with the `app_name`

Reply:
So there was a string that specifically had %s. But this one (and few others) didn't. I am not sure if this was intentional?

Reply:
This is definitely a **blocking change**. In general, we don't want localizers to be translating our app name. We would typically add a note if we used `Firefox` in a string. https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/main/res/values/strings.xml#337.

This string also exists in AC, which shouldn't need to know anything about Firefox. Please do something like https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/main/res/values/strings.xml#398
```suggestion
    <!-- Subtitle text displayed under the title of the save credit card dialog. The first parameter is the name of the app (e.g. Firefox) -->
    <string name="mozac_feature_prompts_save_credit_card_prompt_body_1">%s encrypts your card number. Your security code won’t be saved.</string>
```

Reply:
Please note I have not added this particular string change of /feature/prompts because it might require additional work (sending down the appName from fenix side) than just changing a string. I didn't want to hold the update to rest of the strings, hence this will be done separately in another PR!